python/py-pynetdicom3/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+github.setup        pydicom pynetdicom3 7ebc172
+name                py-pynetdicom3
+version             1.0.0.dev0.20181211
+python.versions     27 36 37
+platforms           darwin
+license             MIT BSD
+maintainers         {eborisch @eborisch} \
```

Comment:
It's a little easier to do greps for what is and isn't openmaintainer when it's on the same line.